veracruz-utils/src/policy/policy.rs

```diff
@@ -336,24 +262,15 @@ impl Policy {
     /// Returns the identity of any principal in the computation who is capable
     /// of requesting a shutdown of the computation.  At the moment, only the
     /// principals who can request the result can also request shutdown.
-    pub fn expected_shutdown_list(&self) -> Vec<u32> {
+    pub fn expected_shutdown_list(&self) -> Vec<u64> {
```

Comment:
I think in the runtime (in fact, I believe in other components, u64 are more common) all the clients are identified by u64 instead of u32, while the Json parser needs an integer to be u32. Hence use the function to reduce type conversion in the runtime.